migrator/README.md

```diff
@@ -1,23 +1,98 @@
 # StackRox Database migration
 
+## Purpose of database migrations
```

Comment:
Great write up.  this should prove very useful going forward.

Reply:
Couldn't agree more!